playbook/app/pb_kits/playbook/pb_pill/docs/_pill_variants.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react'
+import Pill from '../_pill.jsx'
+
+const PillVariants = () => {
+  return (
+    <div>
+      <Pill
+          text="success"
+          variant="success"
+      />
```

Comment:
I believe you need to pass global props to the kit so the dark mode is retained. Click the dark mode button to check this. Also look at another react example 

```suggestion
const PillVariants = (props) => {
  return (
    <div>
      <Pill
          text="success"
          variant="success"
          {...props}
      />
```